content/en/threat-model.md

```diff
@@ -0,0 +1,141 @@
+|                        |                  | x             | x                     | x            |               |              |             | Same as [Fulcio CT log + Rekor server] | Replay and fork attacks will not be reported by monitors.                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|                        | x                |               |                       |              |               |              |             | Arbitrary software attack: Can issue certs for any OIDC issuer/identity and use those to sign any software desired. Bob will see these certificates as valid as they are signed by the Fulcio CA and included in the Fulcio CT log. | Fulcio CT log prevents changes to the history  or blocking issuance / modification of past things (which is not that relevant given the ability to issue keys, etc. now) To recover, the TUF root of trust for Fulcio needs to be invoked to revoke trust in the compromised Fulcio CA server.    |
+|                        | x                | x             |                       |              |               |              |             | Arbitrary software attack: Same as Fulcio CA server, but can now also rewrite historical Fulcio certificates in the Fulcio CT log by changing the Merkle tree hash so that the times will appear different when Bob queries. | Modifying CT log history would be caught by the Fulcio monitor. Can recover using the Sigstore root of trust.                                                                                                                                                                                                               |
+|                        | x                |               |                       | x            |               |              |             | Fulcio CA server + Rekor compromise capabilities. | The recovery is harder than just Fulcio CA server or Rekor server compromise because the history of both tables may be lost.                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
```suggestion
|                        | x                |               |                       | x            |               |              |             | Fulcio CA server and Rekor compromise capabilities. | The recovery is harder than just Fulcio CA server or Rekor server compromise because the history of both tables may be lost.                                                                                                                                                                                                                                                                                                                                                                           |
```